libsawtooth/src/client/rest.rs

```diff
@@ -89,6 +89,42 @@ impl SawtoothClient for RestApiSawtoothClient {
             |item: Result<Batch, _>| item.map(|batch| batch.into()),
         )))
     }
+    /// Get the transaction with the given transaction_id from the current blockchain
+    fn get_transaction(
+        &self,
+        transaction_id: String,
+    ) -> Result<Option<ClientTransaction>, SawtoothClientError> {
+        let url = format!("{}/transactions/{}", &self.url, &transaction_id);
+        let request = Client::new().get(&url);
```

Comment:
Now that you've added `Single<T>`, this block may be generalizable, like `PagingIter<T>`

Reply:
I made a generic type 'get' function to be used in `get_batch` and `get_transaction`